.drone.yml

```diff
@@ -0,0 +1,37 @@
+pipeline:
+  build_6:
+    image: node:6
+    commands:
+      - npm --loglevel warn install
+      - npm run lint
+      - npm run test
+    when:
+      event: [push, pull_request]
```

Comment:
is this running tests on `push` against what? The app builds on `pull_request`s. Or am I missing something?